immuni_exposure_ingestion/core/config.py

```diff
@@ -93,8 +93,15 @@
 BATCH_PERIODICITY_CRONTAB: str = config(
     "BATCH_PERIODICITY_CRONTAB",
     cast=validate_crontab("BATCH_PERIODICITY_CRONTAB"),
-    default="0 0 * * *",
+    default="*/5 * * * *",
```

Comment:
I'd leave the default as it should work in production. If you need to run them more frequently while testing, I'd change the injected environment variable on the testing infrastructure.
We do the same in staging where we use `* * * * *`.